backend/app/assets/tailwind/application.css

```
@@ -1 +1,40 @@
 @import "tailwindcss";
+
+@theme {
```

Stylelint (17.9.0)

[error] Unexpected unknown at-rule "@theme" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)